example/cumulus-tf/main.tf

```diff
@@ -1,13 +1,17 @@
 terraform {
   required_providers {
-    aws  = ">= 2.31.0"
+    aws  = ">= 3.5.0"
```

Comment:
I'm assuming this is required for the ignore_tags block?

Reply:
Yup